projects/storefrontlib/shared/components/chat-messaging/messaging/messaging.component.html

```diff
@@ -0,0 +1,80 @@
+<ng-container *ngIf="messageDetails$ | async as messageDetails">
+  <div class="container">
+    <div class="cx-avatar-line"></div>
+    <div class="cx-messages" id="cx-messages">
+      <ng-container *ngFor="let event of messageDetails.messages">
+        <div
+          class="cx-message-card"
+          [ngClass]="{ 'cx-message-display': event.author !== 'Mark Rivers' }"
```

Comment:
🔴  (hardcoded name) even if it's using mocks

Reply:
it will be handled with the flag that matches the author with current user